yt_dlp/extractor/eplus.py

```diff
@@ -11,7 +11,7 @@
 
 class EplusIbIE(InfoExtractor):
     IE_NAME = 'eplus:inbound'
-    IE_DESC = 'e+ (イープラス). Only works for the link starts with "live.eplus.jp/ex/player?ib="'
+    IE_DESC = 'e+ (イープラス)'
```

Comment:
Well, the eplus player links for local and overseas users may be different. I'm not able to buy tickets as a local, so this extractor, now, is only for extracting info from inbound (ib) links.

Local users will not use this extractor if they see that description, so I think it's better to keep the previous text here.

```suggestion
    IE_DESC = 'e+ (イープラス). Only works for the link starts with "live.eplus.jp/ex/player?ib="'
```

Reply:
If the URL does not match, yt-dlp will give "Unsupported URL" error. It's not IE_DESC's job to write out the matching URL patterns. Maybe you'd like to change the description to `e+ overseas` instead

Reply:
Got it, and thanks for your nice suggestion. See 157ee41b6c03364d6ceafbb76c13d2b1767cabfb .